holmes/core/conversations_worker/realtime_manager.py

```
@@ -392,16 +434,16 @@ async def _full_reconnect(self) -> bool:
         except Exception:
-            logging.exception(
-                "Failed to re-sign-in to Supabase before reconnect",
+            logging.warning(
+                "Failed to re-sign-in to Supabase before reconnect; will retry",
                 exc_info=True,
             )
             return False
         try:
             await self._connect_and_subscribe()
             return True
         except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)